datalad/support/annexrepo.py

```diff
@@ -63,14 +74,44 @@ def _annex_init(self):
         # not existing paths, etc.
 
         status = self.cmd_call_wrapper.run('cd %s && git annex init' % self.path)
-        if status != 0:
-            datalad.log.lgr.error('git annex init returned status %d.' % status)
+        if status not in [0, None]:
+            lgr.error('git annex init returned status %d.' % status)
 
 
-    def dummy_annex_command(self):
-        """Just a dummy
+    def annex_get(self, pattern, **kwargs):
+        """Get the actual content of files
 
-        No params, nothing to explain, should raise NotImplementedError.
+        Parameters:
+        -----------
+        pattern: str
+            glob pattern defining what files to get
 
+        kwargs: options for the git annex get command. For example `from='myremote'` translates to annex option
+            "--from=myremote"
         """
-        raise NotImplementedError
+
+        paths = glob.glob(pattern)
+        #TODO: regexp + may be ext. glob zsh-style
+
+        pathlist = ''
+        for path in paths:
+            pathlist += ' ' + path
```

Comment:
and that is where I thought to raise an issue of ... files with spaces in the names.  You could extend on of the test repos for testing it or just generate using @with_tree

and above loop is identical to ```pathlist = ' '.join(paths)```  but once again we might need at least to guard for spaces... and may be some other interesting characters since it is possible to ahve files with ' or " in the filenames... testcase could help to iron out working use of wrapping etc